packages/backend/src/profile.test.ts

```diff
@@ -99,26 +126,20 @@ describe('Profile', () => {
                 signature,
             };
 
-            const { status } = await request(app)
+            const response = await request(app)
                 .post(`/${wallet.address}`)
                 .send(signedUserProfile);
 
+            // log status message
+            console.log(response.info);
```

Comment:
Do we need this log here?  Otherwise we can remove it to keep the logs clean

Reply:
true, removed